doc/lua_api.md

```diff
@@ -6155,13 +6155,16 @@ Environment access
     * Returns the position of the blocking node when `false`
     * `pos1`: First position
     * `pos2`: Second position
-* `minetest.raycast(pos1, pos2, objects, liquids)`: returns `Raycast`
+* `minetest.raycast(pos1, pos2, objects, liquids, pointabilities)`: returns `Raycast`
     * Creates a `Raycast` object.
     * `pos1`: start of the ray
     * `pos2`: end of the ray
     * `objects`: if false, only nodes will be returned. Default is `true`.
     * `liquids`: if false, liquid nodes (`liquidtype ~= "none"`) won't be
                  returned. Default is `false`.
+    * `pointabilities`: Contains lists to override the `pointable` property of
+      nodes and objects. Uses the same format as the `pointabilities` property
+      of item definitions. Default is `nil`.
```

Comment:
clearer imo
```suggestion
    * `pointabilities`: Allows overriding the `pointable` property of
      nodes and objects. Uses the same format as the `pointabilities` property
      in the item definition. Default is `nil`.
```

Reply:
```suggestion
    * `pointabilities`: Allows overriding the `pointable` property of
      nodes and objects. Uses the same format as the `pointabilities` property
      of item definitions. Default is `nil`.
```

"the item definition" seems to imply that there is one currently relevant item definition, which there isn't